### ISSUE TITLE
Missing SyncBatchNormalization in keras core

### ISSUE THREAD

**Issue by edwardyehuang** [keras-team/keras#18667]
There is no implementation of `SyncBatchNormalization` in keras core. Note that, in TensorFlow 2.14, `tf.keras.layers.BatchNormalization` has an argument `synchronized` to replace `SyncBatchNormalization`, but it is also missing in `layers.BatchNormalization` of keras core

**Comment by fchollet** [comment]
The fix should be to implement the `synchronized` argument in Keras 3. Are you able to open a PR for this?

**Comment by edwardyehuang** [comment]
> The fix should be to implement the `synchronized` argument in Keras 3. Are you able to open a PR for this?

PR #18671  (only for TensorFlow)
